bitnami/haproxy/templates/service-account.yaml

```diff
@@ -1,13 +1,21 @@
+{{- if .Values.serviceAccount.create }}
```

Comment:
I would keep just this condition and remove the unrelated changes

Reply:
Hi @pablogalegoc Thanks for your feedback, first time contributing to a project. 

I have updated the PR accordingly

Reply:
Hey @GGre4, that's awesome! This kind of little changes are very useful and help building confidence.
I asked to keep the changes just to what's intended for the PR because then it's easier to find them and reason about them 🙂